### PR TITLE
ci: cache package to reduce build time

### PR DESCRIPTION
**Pull request by kenhys**

### CODE REVIEW

**Comment by kenhys** [comment]
rpm cached:
https://github.com/fluent/fluent-package-builder/actions/runs/11586846546/job/32259162192

deb cached:
https://github.com/fluent/fluent-package-builder/actions/runs/11586846546/job/32259162421

**Comment by daipom** [comment]
Thanks! 
I think we should add this to the master branch because this is not directly related to this feature branch.

File: .github/workflows/apt.yml [hunk 1]
```diff
@@ -46,7 +46,17 @@ jobs:
...
           sudo gem uninstall fileutils
+      - name: cache deb
+        uses: actions/cache@v4
+        id: cache-deb
+        with:
+          path: |
+            fluent-package/apt/repositories
+            fluent-apt-source/apt/repositories
+            fluent-lts-apt-source/apt/repositories
+          key: ${{ runner.os }}-cache-${{ matrix.rake-job }}-${{ hashFiles('**/config.rb', '**/Rakefile', '**/Gemfile*', '**/deb/*') }}
```
Comment (daipom) [inline .github/workflows/apt.yml:57]: Using `hashFiles` is an excellent idea.
but is this enough? Don't we need to include more files?

* `fluent-package/templates/**`
* `fluent-package/debian/**`
* files under `fluent-package/apt/` except for test files, such as Dockerfile

Comment (kenhys) [reply .github/workflows/apt.yml]: Thanks, take into account more files .

**Comment by kenhys** [comment]
> I think we should add this to the master branch because this is not directly related to this feature branch.

Basically so, but it is effective to apply to this feature branch.

**Comment by kenhys** [comment]
re-target to master.

**Pull request closed without merging**